public/js/jibe.js

```diff
@@ -399,6 +408,39 @@ var Jibe = (function (BCSocket, CodeMirror, Replay, showdown, Timestamps, TextFo
       // flag current version
       api.flagVersion ();
     });
+
+    // Table of Contents stuff
+    /* Toggles table of contents panel
+    */
+    $('#toggle-toc').click(function(){
+      if( $('#toc-container').is(":visible")){
+        $('#toc-container').hide('slow',function(){
+          $('#editor-preview-container').removeClass("col-md-9").promise().done(function(){});
```

Comment:
It's waiting until the hide animation is completed before changing the size of the editor div.